roles/users/tasks/main.yml

```diff
@@ -11,12 +11,16 @@
   delay: 10
   # perform_users_role is True by default so no need to fail the play if there's an error.
   ignore_errors: true
+  tags:
+    - pubkeys
```

Comment:
Did you mean to commit the changes in this file?

Reply:
I did yeah; I was having problems getting `--skip-tags user,pubkeys` to work properly without this

Reply:
Weird.  What was the full command you were running?

Reply:
Something like `ansible-playbook -v -i ~/dev/ceph-sepia-secrets/ansible/inventory/sepia -l 'paddles.front.*' ./paddles.yml --skip-tags user,pubkeys`